sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpSender.cs

```diff
@@ -210,12 +209,15 @@ internal class AmqpSender : TransportSender
             ServiceBusMessageBatch messageBatch,
             CancellationToken cancellationToken)
         {
-            AmqpMessage messageFactory() => AmqpMessageConverter.BatchSBMessagesAsAmqpMessage(messageBatch.AsEnumerable<ServiceBusMessage>());
-            await _retryPolicy.RunOperation(async (timeout) =>
-                await SendBatchInternalAsync(
-                    messageFactory,
-                    timeout,
-                    cancellationToken).ConfigureAwait(false),
+            await _retryPolicy.RunOperation(static async (value, timeout, token) =>
+                {
+                    var (sender, msgs) = value;
```

Comment:
nit:  Can we go with `messages` so that the context is more clear?  Here and below....   😄

Reply:
Same comment as abobeu